src/Microsoft.DotNet.XHarness.iOS.Shared/SdkVersions.cs

```diff
@@ -5,11 +5,11 @@
 /// </summary>
 public static class SdkVersions
 {
-    public static string Xcode { get; private set; } = "13.3";
-    public static string OSX { get; private set; } = "12.00";
-    public static string iOS { get; private set; } = "15.4";
+    public static string Xcode { get; private set; } = "14.3";
```

Comment:
looks like this isn't used, why do we need this?

Reply:
and does this mean we'd require newest versions of Xcode or iOS Simulators? just thinking since we don't have branches for xharness yet: https://github.com/dotnet/xharness/issues/1010

Reply:
Yeah I think we need to test this with Ventura/Xcode 14.3

Reply:
Actually, I don't think these matter much. I know they were used in xamarin-macios which injects this class but not here